app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+    respond_to :html, :json
```

Comment:
Can't we use respond_to in the actions of every controller?

Reply:
getting format error in mailCatcher, this resolved the issue for me

Reply:
resolved